### PR TITLE
Endpoint for listing malfunctions

### LINKED ISSUE #57: Create endpoint for listing malfunctions
Includes adding malfunctions to the database

### PR DESCRIPTION
**Pull request by jankupis**
Closes #57

### CODE REVIEW

File: BikeRentalApi/endpoints/malfunctionsList.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from django.http import JsonResponse
+from rest_framework import status
+
+from BikeRentalApi.decorators.roleRequired import RoleRequired
+from BikeRentalApi.enums import Role
+from BikeRentalApi.models import Malfunction
+# POST: create bikes
+# GET: list all bikes
```
Comment (debskipawel) [inline BikeRentalApi/endpoints/malfunctionsList.py:8]: I don't think these comments belong here. Please remove them or change them and put them in line 1

File: BikeRentalApi/serializers/stationSerializer.py [hunk 2]
```diff
@@ -7,16 +7,13 @@
 
 class StationSerializer(serializers.ModelSerializer):
     id = CharField()
-    status = SerializerMethodField()
+    status = CharField(source = 'state.label')
```
Comment (debskipawel) [inline BikeRentalApi/serializers/stationSerializer.py:10]: I really like this, nice change.

**Review by debskipawel — CHANGES_REQUESTED** [review]
Looks really solid, just change this one thing with comments and it's good to go! 🎉

File: BikeRentalApi/serializers/stationSerializer.py (hunk 2 above)
Comment (jankupis) [reply BikeRentalApi/serializers/stationSerializer.py:10]: Thanks!

File: BikeRentalApi/endpoints/malfunctionsList.py (hunk 1 above)
Comment (jankupis) [reply BikeRentalApi/endpoints/malfunctionsList.py]: Fixed

**Review by debskipawel — APPROVED** [review]
Looks great! 😎

**Pull request merged**